src/quality/quality.js

```diff
@@ -35,7 +35,12 @@ Object.assign(mejs.MepDefaults, {
 	/**
 	 * @type Function
 	 */
-	qualityChangeCallback: null
+	qualityChangeCallback: null,
+	/**
+	 * The path where the icon sprite is located
+	 * @type {String}
+	 */
+	iconSpritePath: 'mejs-quality.svg',
```

Comment:
In this case, this isn't a sprite, because there are not multiple symbol in the same svg.

Reply:
Description and name has been updated accordingly.

Reply:
Thanks